test/stores/widgets/StopGapWidgetDriver-test.ts

```diff
@@ -616,4 +617,32 @@ describe("StopGapWidgetDriver", () => {
             expect(client.uploadContent).toHaveBeenCalledWith("data");
         });
     });
+
+    describe("downloadFile", () => {
+        let driver: WidgetDriver;
+
+        beforeEach(() => {
+            driver = mkDefaultDriver();
+        });
+
+        it("should download a file and return the blob", async () => {
+            // eslint-disable-next-line no-restricted-properties
+            client.mxcUrlToHttp.mockImplementation((mxcUrl) => {
+                if (mxcUrl === "mxc://example.com/test_file") {
+                    return "https://example.com/_matrix/media/v3/download/test_file";
+                }
+
+                return null;
+            });
+
+            fetchMockJest.get("https://example.com/_matrix/media/v3/download/test_file", "test contents");
```

Comment:
wouldn't this be
```suggestion
                    return "https://example.com/_matrix/media/v3/download/example.com/test_file";
                }

                return null;
            });

            fetchMockJest.get("https://example.com/_matrix/media/v3/download/example.com/test_file", "test contents");
```

it might not be obvious why this calls the deprecated API, but maintainers will know better than me.

Reply:
> wouldn't this be

Done

> it might not be obvious why this calls the deprecated API, but maintainers will know better than me.

The implementation relies on [`Media`](https://github.com/matrix-org/matrix-react-sdk/blob/ec829435bfe4ab8010740fc6884571e23f6e652e/src/customisations/Media.ts#L36) . Not sure if the test should mock it away or not :shrug: